playground/frontend/playground_components/lib/src/theme/theme.dart

```diff
@@ -32,16 +33,21 @@ class BeamThemeExtension extends ThemeExtension<BeamThemeExtension> {
   final Color lightGreyBackgroundTextColor;
   final Color secondaryBackgroundColor;
 
+  final Color codeBackgroundColor;
   final TextStyle codeRootStyle;
   final CodeThemeData codeTheme;
 
+  final MarkdownStyleSheet? markdownStyle;
```

Comment:
Any reasons to make it nullable?

Reply:
There's a comment: `// TODO(nausharipov): markdownStyle is nullable, because I don't know what to set here`